### ISSUE TITLE
[Qwen2.5-VL] grad-norm

### ISSUE THREAD

**Issue by hyunjun-eun** [alibaba/Pai-Megatron-Patch#579]
We trained qwen2.5-vl, but checked large high grad-norm (over 100) while training. Although grad-norm is too large, lm loss seems to be decreased nicely. Is it right in training of qwen2.5-vl? (We know that grad-norm is generally under 1.0.)

![Image](https://github.com/user-attachments/assets/e4b9ecf8-5d70-4069-a133-dee0cda64abd)
![Image](https://github.com/user-attachments/assets/c78a5000-d08f-435f-84be-4a6d62e47868)

**Comment by EthanChen1234** [comment]
Hi, I also encountered the same problem. The clip-grad 1.0 makes the loss nicely.

"(We know that grad-norm is generally under 1.0.)", could you give me the source paper?

I trained the qwen2.5-0.5B and qwen2.5-1.5B from scratch, the grad norm is around 2500 when training 100k steps

**Comment by oceanos74** [comment]
@EthanChen1234 What was the gradient clipping value in those experiments? Did you get your training done well?

If grad norm was around 2500 with clipping value of '1', gradient was scaled down by ~1/2500 and it leads to similar effect of 1/2500 smaller learning rate setting. I wonder the model was trained well with these small gradient estimates or learning rate setting.

**Comment by tiggerwu** [comment]
I also have this problem. When I trained Qwen2-1.5B, I found grad-norm is about ~1000. I think it's because the loss is not divided by seq_len. I don't know whether it's correct or not.
